### ISSUE TITLE
Issue with "Unable to require openssl...." when installing Rails - Fix

### ISSUE THREAD

**Issue by putraisyraq** [TheOdinProject/curriculum#5839]
Hi,
I am a new learner. I stumble upon the `Unable to require openssl....` error while trying to install rails few days ago. I googled everywhere for fix to the point I was so sure I made a mistake in terminal and even reformatted my mac. However problem still persists.

After few more googling, I found this article [here (look for pinkasey)](http://stackoverflow.com/questions/20309145/unable-to-require-openssl-when-initialling-rails#24347586) and then read more in the linked page, https://github.com/rvm/rvm/issues/3923 (fix provided by mpapis, thank him!). The fix helped!

in short, go to terminal, key in:
`rvm get branch /bugfix/3923_set_osx_brew_openssl_compiler_config`
and then `rvm install 2.4.0` and `gem install rails`

**Comment by KevinMulhern** [comment]
Hey @putraisyraq, thanks for opening this and thank you for providing the solution to the problem 😄 

We've had quite a few people in the chat with the same problem this week. Hopefully rvm release a fix to the master branch soon.

**Comment by putraisyraq** [comment]
You're welcome. Just helping others like me. I hope it gets updated soon too. :)

**Comment by KevinMulhern** [comment]
Havent had any reports of this in about a week so I think this is safe to close. Thanks Again @putraisyraq
